conf/initializers/template.go

```diff
@@ -39,6 +44,12 @@ func toTimeAgo(timestamp time.Time) string {
 	return timeago.FromTime(timestamp)
 }
 
+func toTimeStamp(timestamp time.Time) string {
```

Comment:
Olivier [suggested](https://github.com/junan/google-scraper/pull/34#discussion_r596585378) to move these custom functions  view helpers in my PR as these functions are related to view not initializer(mapping code `AddFuncMap` can say in this initializer I think). How about to do that?

Reply:
https://github.com/Lahphim/go-crawler-challenge/pull/64/commits/4dff1ce9d5a751dbac49ae6bc6ca36b43aeffbd6